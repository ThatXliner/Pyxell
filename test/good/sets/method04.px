
x = {'a'..'z'}
{Char} y = {}

x.remove('a')
x.remove('t')

n = 0
while x.length > 0 do
    n += 1
    y.add(x.pop())

print x
print n
print y == {'b'..'z'} - {'t'}

y.clear()
print y.length
