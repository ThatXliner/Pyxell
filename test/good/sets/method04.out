{}
24
true
0
